packages/@orchidui-vue/src/Chip/OCChip.vue

```diff
@@ -1,11 +1,48 @@
 <template>
-  <span>
-    {{ label }}
+  <span class="rounded-full py-1 px-2 text-xs" :class="className">
+    <template v-if="label">{{ label }}</template>
+    <slot v-else />
   </span>
 </template>
-<script setup>
-import { defineProps } from "vue";
-defineProps({
-  label: String,
-});
-</script>
+<script>
+export default {
+  props: {
+    color: {
+      type: String,
+      default: "blue",
+    },
+    label: {
+      type: String,
+    },
+  },
+  computed: {
+    className() {
+      let className = "";
+      switch (this.color) {
```

Comment:
The `switch` is unreadable in this case and the switch is an antipattern in most cases. It would be better to rewrite this code like this:
```
const ColorClasses = {
  "purple": "bg-hit-purple-50 text-hit-purple ",
  "orange": "bg-hit-warning-50 text-hit-warning ",
  "red": "bg-hit-red-50 text-hit-red ",
  "dark-blue": "bg-hit-primary-50 text-hit-primary ",
  "grey": "bg-hit-grey-100 text-hit-grey-700 ",
  "green": "bg-hit-success-50 text-hit-success ",
}
return ColorClasses[this.color] || "bg-hit-blue-50 text-hit-blue";
```
Here you can see more examples of how you can replace switch: https://github.com/HowProgrammingWorks/Antipatterns/blob/master/JavaScript/02-Procedural/3-switch.js